src/components/layout/nav.js

```diff
@@ -1114,6 +1137,94 @@ export const NavPartners = ({ no_login_signup }) => {
     )
 }
 
+// Note: When using layout component for security page, please add type='security' and padding_top='10rem'
+export const NavSecurity = () => {
+    const nav_ref = useRef(null)
+    const button_ref = useRef(null)
+
+    return (
+        <>
+            <NavWrapperPartners ref={nav_ref}>
+                <DerivHomeWrapper>
+                    <HomeContainer justify="space-between">
+                        <StyledContainer justify="flex-start">
+                            <HomeLink to="/">
+                                <Header weight="normal" color="grey-19" type="paragraph-2">
+                                    {localize('Go to Deriv.com')}
+                                </Header>
+                            </HomeLink>
+                            <HomeLink to="/story/">
+                                <Header weight="normal" color="grey-19" type="paragraph-2">
+                                    {localize('About us')}
+                                </Header>
+                            </HomeLink>
+                            <HomeLink to="/contact_us/">
+                                <Header weight="normal" color="grey-19" type="paragraph-2">
+                                    {localize('Contact us')}
+                                </Header>
+                            </HomeLink>
+                        </StyledContainer>
+
+                        <Flex ml="auto" ai="center" width="auto">
+                            <LanguageSwitcher short_name="true" security />
+                        </Flex>
+                    </HomeContainer>
+                </DerivHomeWrapper>
+
+                <StyledNavPartners>
+                    <StyledNavWrapper>
+                        <NavLeftPartners>
+                            <NavLogoLink to="/" aria-label={localize('Security')}>
+                                <img src={LogoSecurity} alt="logo security" />
+                            </NavLogoLink>
+                        </NavLeftPartners>
+                        <SecurityNavRight button_ref={button_ref} mounted={true}>
+                            <LinkButton
+                                to={'mailto:security@deriv.com'}
+                                is_mail_link
+                                external="true"
+                                target="_blank"
+                                tertiary
+                                style={{ width: '16rem' }}
+                            >
+                                <span>{localize('Submit a report')}</span>
+                            </LinkButton>
+                        </SecurityNavRight>
+
+                        <Mobile>
+                            <Flex ai="center" jc="space-between">
+                                <LogoLinkMobile
+                                    style={{ margin: 'unset' }}
```

Comment:
You can define the margin style in the component styles above. Keep them together for easier future debugging.